From ca34590e5d0065db581b3b4c05c9c7aea0015dff Mon Sep 17 00:00:00 2001
From: Michael Forney <mforney@mforney.org>
Date: Fri, 14 Jun 2019 12:40:56 -0700
Subject: [PATCH] rsync: Add missing includes

- stdio.h in socket.c for sscanf
- sys/types.h in extern.h for various type definitions
---
 usr.bin/rsync/extern.h | 2 ++
 usr.bin/rsync/socket.c | 1 +
 2 files changed, 3 insertions(+)

diff --git a/usr.bin/rsync/extern.h b/usr.bin/rsync/extern.h
index 305821be579..040588003a4 100644
--- a/usr.bin/rsync/extern.h
+++ b/usr.bin/rsync/extern.h
@@ -17,6 +17,8 @@
 #ifndef EXTERN_H
 #define EXTERN_H
 
+#include <sys/types.h>
+
 /*
  * This is the rsync protocol version that we support.
  */
diff --git a/usr.bin/rsync/socket.c b/usr.bin/rsync/socket.c
index 36384d063a0..43d8d7d01af 100644
--- a/usr.bin/rsync/socket.c
+++ b/usr.bin/rsync/socket.c
@@ -28,6 +28,7 @@
 #include <poll.h>
 #include <resolv.h>
 #include <stdlib.h>
+#include <stdio.h>
 #include <string.h>
 #include <unistd.h>
 #include <err.h>
-- 
2.22.0

